### PR TITLE
Hex package

### PR DESCRIPTION
**Pull request by Licenser**
Hi I've published riak_dt as a hex package, this are the changes necessary to make it hex-able with riak3. Since it is your project it would be great if you'd take over the package, for that you can ask hex.pm to change ownership, until then I'll try to keep it up to date with the latest tagged release.

### CODE REVIEW

**Comment by zeeshanlakhani** [comment]
Thanks @Licenser... guessing you mean rebar3 :). @russelldb and I will discuss... still haven't made the move to rebar3 w/ this, but we should.

**Comment by Licenser** [comment]
argh yes sorry rebar3 not riak3 ;) freudian slip there ...

So it doesn't need rebar3 to build or anything just as there is no hex upload option with anything else, that's all that requires rebar3 in it.

**Comment by Licenser** [comment]
oh, slightly unrelated, I only listed myself as a co-maintainer in there so you don't get bothered by people having questions about the package. I'm not sure if the maintainers are the package maintainers or the code maintainers, just drop that if you take over the package :)

**Comment by cmeiklejohn** [comment]
:+1: on moving to rebar3, please.

**Comment by zeeshanlakhani** [comment]
@Licenser of course, but as @cmeiklejohn pointed out... would be nice :). And, again, thank you :).

File: src/riak_dt.app.src [hunk 1]
```diff
@@ -1,7 +1,7 @@
 %% -*- erlang -*-
 {application, riak_dt,
  [
-  {description, ""},
+  {description, "riak CTDT datatypes"},
```
Comment (russelldb) [inline src/riak_dt.app.src]: CRDT

Comment (zeeshanlakhani) [inline src/riak_dt.app.src]: nice catch :).

**Review by russelldb — COMMENTED** [review]
All I have to do is merge this and then we are hexable? @Licenser can you help me learn how to maintain the hex package?

**Comment by Licenser** [comment]
neva!

Ah well OK, but just because it's you russell ;)!

Yes since you don't have deps this is really all that's needed.
- Well nearly, you need a hex.pm account and I'll add you as a maintainer (and then remove myself).
- Also to publish the `app.src` file needs a 'correct' version in it, last I checked the hex plugin doesn't handle git versioning.
- you need to add the rebar3 plugin to your global plugins in rebar

```
$ cat ~/.config/rebar3/rebar.config
{plugins, [rebar3_hex]}.
```

But other then that it'll just be `rebar3 hex publish` and it sends it up.

**Comment by russelldb** [comment]
Thanks @Licenser. When I get back to UK I'll do it. Promise! Thanks for your patience and help.

**Comment by Licenser** [comment]
hey no worries at all, riak_dt helps me soooo much in my stuff it's really the least I can do.

**Comment by russelldb** [comment]
+1

**Pull request merged**